### PR TITLE
Feat/category

### PR DESCRIPTION
**Pull request by Debarshi95**
PR for category and individual video page
Preview:https://6270cb7d847fba736cd4609f--debarshib-wetube.netlify.app/

### CODE REVIEW

File: src/components/Categorybar/Categorybar.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  min-width: 8rem;
+  margin: 0.75rem 0.5rem;
+}
+.Categorybar__link > a:hover {
+  background-color: var(--color-red-hover);
+}
+
+@media screen and (max-width: 422px) {
+  .Categorybar__link > a {
+    padding: 0.2rem 0.7rem;
```
Comment (tarunsankhla) [inline src/components/Categorybar/Categorybar.css]: Avoid using such minor decimal like 0.2rem instead use pixels

File: src/components/SideDrawer/SideDrawer.js [hunk 2]
```diff
@@ -12,11 +12,11 @@ const links = [
...
     pathname: '/',
   },
-  {
-    name: 'Explore',
-    icon: <AiOutlineHome size="1.75rem" color="inherit" />,
-    pathname: '/explore',
-  },
+  // {
+  //   name: 'Explore',
+  //   icon: <AiOutlineHome size="1.75rem" color="inherit" />,
```
Comment (tarunsankhla) [inline src/components/SideDrawer/SideDrawer.js:17]: Remove commented code

File: src/index.js [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
   <React.StrictMode>
-    <ApolloProvider>
-      <SideDrawerProvider>
-        <App />
-      </SideDrawerProvider>
-    </ApolloProvider>
-  </React.StrictMode>
+    <ErrorBoundary>
+      <ApolloProvider>
+        <SideDrawerProvider>
```
Comment (tarunsankhla) [inline src/index.js:13]: Suggestion create file with all providers,and only import once here

File: src/pages/App.js [hunk 4]
```diff
@@ -11,13 +14,42 @@ const App = () => {
...
+            element={
+              <Suspense fallback={<Loader />}>
+                <LazyCategory />
+              </Suspense>
+            }
+          />
+        </Route>
+        <Route
+          path="/video/:name"
           index
```
Comment (tarunsankhla) [inline src/pages/App.js:45]: Can create const of parh

File: src/pages/Video/Video.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+          {videoData?.videos &&
+            videoData.videos.slice(5, 12).map((video) => (
+              <Card
+                className="Video__card"
+                key={video.id}
+                item={video}
+                imgProps={{
+                  width: '100%',
+                  height: lg ? '20rem' : '13rem',
```
Comment (tarunsankhla) [inline src/pages/Video/Video.js]: Avoid using const like this

**Review by tarunsankhla — APPROVED** [review]
Overall great just minor changes

File: src/pages/Video/Video.css [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+}
+.Video__card {
+  width: 24rem;
+}
+
+@media screen and (max-width: 1068px) {
+  .Video__root {
+    margin: 0;
+    padding: 0 1.5rem;
+    max-width: 100%;
```
Comment (SinchanaMS) [inline src/pages/Video/Video.css]: Keep consistent spacing after each block.

File: src/pages/Category/Category.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
+import { upperFirst } from 'lodash';
+import { useQuery } from '@apollo/client';
+import { Card, Loader } from 'components';
+import { FETCH_VIDEO_BY_CATEGORY } from 'constants/queries/queries';
+import React from 'react';
```
Comment (SinchanaMS) [inline src/pages/Category/Category.js]: This import isn't required here, you can remove it.

**Review by SinchanaMS — APPROVED** [review]
The code looks fine. A suggestion for the UI aspect though - the video title is not completely visible on the video page. Please look into it.
image

**Pull request merged**